mpas_analysis/shared/html/pages.py

```diff
@@ -74,7 +74,11 @@ def generate_html(config, analyses, refConfig=None):  # {{{
         page.add_component(componentName, component.subdirectory,
                            firstImageFileName)
 
-    page.generate()  # }}}
+    page.generate()
+
+    print("Done.")
```

Comment:
I got annoyed as well that it prints "Generating webpage..." but never says it finished.